runtime/stdlib/contract_update_validation.go

```diff
@@ -16,19 +16,21 @@
  * limitations under the License.
  */
 
-package runtime
+package stdlib
```

Comment:
Not sure if stdlib is the correct place for this

Reply:
Yeah, it isn't really "standard library" code, but it is needed by the standard library's account contract deployment/update code, and I could not think of another place. Do you have any ideas where it could live instead?